deployment/distribution/build.gradle

```diff
@@ -24,8 +25,14 @@ applicationDistribution.from('src/main/resources/') {
     into 'conf'
 }
 
-applicationDistribution.from(project(':plugins:ddf-plugins').file('build/libs')) {
-    into 'plugins'
+project(':external:implementations').getSubprojects().each{ subProject ->
```

Comment:
✏️ Looks like this file needs to be formatted.

Reply:
Fixed.